libexec/mcd/mcd---spot-ilk

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 set -e
 [[ $MCD_SPOT_ILK ]] ||
-  export MCD_SPOT_ILK=$(seth call ${MCD_SPOT?} 'ilks(bytes32)(address, uint256)' $1);
-echo $MCD_SPOT_ILK
+  MCD_SPOT_ILK=$(seth call ${MCD_SPOT?} 'ilks(bytes32)(address, uint256)' $1);
```

Comment:
I think it is missing the quotes here.

Reply:
I added quotes here, I think this one is ok, but we have to be careful with some of these as the response is a bash array and if we quote it, it will be evaluated as a string elsewhere.